src/selectors/entities/channels.test.js

```diff
@@ -9,6 +9,884 @@ import {sortChannelsByDisplayName, getDirectChannelName} from 'utils/channel_uti
 import * as Selectors from 'selectors/entities/channels';
 import {General, Preferences} from '../../constants';
 
+const sortUsernames = (a, b) => a.localeCompare(b, General.DEFAULT_LOCALE, {numeric: true});
+
+describe('Selectors.Channels.getChannelsInCurrentTeam', () => {
+    const team1 = TestHelper.fakeTeamWithId();
+    const team2 = TestHelper.fakeTeamWithId();
+
+    it('should return channels in current team', () => {
+        const user = TestHelper.fakeUserWithId();
+
+        const profiles = {};
+        profiles[user.id] = user;
+
+        const channel1 = TestHelper.fakeChannelWithId(team1.id);
+        channel1.display_name = 'Channel Name';
+        const channel2 = TestHelper.fakeChannelWithId(team2.id);
+        channel2.display_name = 'Channel Name';
+        const channel3 = TestHelper.fakeChannelWithId(team1.id);
+        channel3.display_name = 'Channel Name';
+        const channel4 = TestHelper.fakeChannelWithId('');
+        channel4.display_name = 'Channel Name';
+
+        const channels = {};
```

Comment:
Feel free to keep doing this how you're doing it, but you could avoid repeating the name of the object if you wanted to
```javascript
const channels = {
    [channel1.id]: channel1,
    [channel2.id]: channel2,
    [channel3.id]: channel3,
    [channel4.id]: channel4,
};
```

Reply:
I was mainly following the patter they were using on the original test. I can do it this other way without any problem.